frontend/src/pages/ResetPasswordEmail.tsx

```diff
@@ -0,0 +1,73 @@
+import { Link, useHistory, useParams } from 'react-router-dom';
+import { Trans } from 'react-i18next';
+import classNames from 'classnames';
+import { ModalContent } from '../components/modals/modalparts/ModalContent';
+import { ModalHeader } from '../components/modals/modalparts/ModalHeader';
+import { ReactComponent as CheckIcon } from '../icons/check_icon.svg';
+import css from './ResetPasswordEmail.module.scss';
+
+const classes = classNames.bind(css);
+
+export const ResetPasswordEmail = () => {
```

Comment:
I don't think we need the "show email in browser" option. Or do we?

Anyway, I would rather have this as a separate pr.